package.json

```diff
@@ -650,6 +650,16 @@
                 "command": "explorer.newFolder",
                 "when": "filesExplorerFocus && !inputFocus"
             },
+            {
+                "key": "c",
+                "command": "filesExplorer.copy",
+                "when": "explorerViewletVisible && filesExplorerFocus && !explorerResourceIsRoot && !inputFocus"
```

Comment:
would `yy` make sense for this? why are we using nvim-tree as the source of conventions instead of more vim-like conventions?

Reply:
It's the most popular file explorer for Neovim, it is not? We have a `y` mapping for copy already. `nvim-tree` allows to use either `y` or `c`, I'm used to `c` on the file explorer, so others might, too.

Reply:
> It's the most popular file explorer for Neovim, it is not?

I have no idea, I've never used it. And before the NERDTree was the most popular for 10+ years, but that makes no difference.  Popular plugins come and go and `c` is the vim convention for "change", the complete opposite of yank.